crates/nu-command/src/platform/ansi/command.rs

```diff
@@ -0,0 +1,354 @@
+use nu_ansi_term::*;
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::Call, engine::Command, Example, IntoPipelineData, PipelineData, ShellError, Signature,
+    SyntaxShape, Value,
+};
+
+#[derive(Clone)]
+pub struct AnsiCommand;
+
+impl Command for AnsiCommand {
+    fn name(&self) -> &str {
+        "ansi"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("ansi")
+            .required(
+                "code",
+                SyntaxShape::String,
+                "the name of the code to use like 'green' or 'reset' to reset the color",
+            )
+            .switch(
+                "escape", // \x1b[
+                "escape sequence without the escape character(s)",
+                Some('e'),
+            )
+            .switch(
+                "osc", // \x1b]
+                "operating system command (ocs) escape sequence without the escape character(s)",
+                Some('o'),
+            )
+    }
```

Comment:
I think we also need a category but not sure which category to use. Just pick one please and run with it, we can always change it later.